external/googleapis/update_libraries.sh

```diff
@@ -54,6 +54,7 @@ declare -A -r LIBRARIES=(
       "@com_google_googleapis//google/iam/credentials/v1:credentials_proto" \
       "@com_google_googleapis//google/iam/admin/v1:admin_proto"
   )"
+  ["ids"]="@com_google_googleapis//google/cloud/ids/v1:ids_cc_grpc"
```

Comment:
I forget where we landed on this. I am just commenting for the sake of commenting.

Reply:
I think we landed on using the `_cc_grpc` form as easier/preferred.